tests/nnstreamer_filter_reload/runTest.sh

```diff
@@ -19,6 +19,51 @@ testInit $1
 
 # NNStreamer and plugins path for test
 PATH_TO_PLUGIN="../../build"
+# This test is valid only with the tensor filter extension for tflite
+if [[ -d $PATH_TO_PLUGIN ]]; then
+    ini_path="${PATH_TO_PLUGIN}/ext/nnstreamer/tensor_filter"
+    if [[ -d ${ini_path} ]]; then
+        check=$(ls ${ini_path} | grep tensorflow-lite.so)
```

Comment:
better to set SO_EXT as you fixed today?

Reply:
Ah:( my bad...